<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dictionary</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgba(245, 245, 245, 179);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_searchWord">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(31, 178, 255);
	selection-background-color: rgb(85, 255, 0);
	padding: 6px;
	border-radius: 10px;
	width:75px;
	height:15px;
	font: bold 8pt &quot;Georgia&quot;;
	
}
QPushButton:hover{
   background-color: rgb(6, 85, 255)
}
QPushButton:pressed {
	border-style:solid;
	border-width:3px;
}</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_ext_dict">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(31, 178, 255);
	selection-background-color: rgb(85, 255, 0);
	padding: 6px;
	border-radius: 10px;
	width:115px;
	height:15px;
	font: bold 8pt &quot;Georgia&quot;;
	
}
QPushButton:hover{
   background-color: rgb(6, 85, 255)
}
QPushButton:pressed {
	border-style:solid;
	border-width:3px;
}</string>
      </property>
      <property name="text">
       <string>Add Dictionaries</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_description">
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 8pt &quot;Georgia&quot;;
	font:bold;
}</string>
      </property>
      <property name="text">
       <string> Description</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_addWord">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(31, 178, 255);
	selection-background-color: rgb(85, 255, 0);
	padding: 6px;
	border-radius: 10px;
	width:75px;
	height:15px;
	font: bold 8pt &quot;Georgia&quot;;
}
QPushButton:hover{
   background-color: rgb(6, 85, 255)
}
QPushButton:pressed{
	border-style:solid;
	border-width:3px;
}</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_version">
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 8pt &quot;Nirmala UI&quot;;
	
}</string>
      </property>
      <property name="text">
       <string>Dictionary Version 1.0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_word">
      <property name="styleSheet">
       <string notr="true">QLabel{
	
	font: 8pt &quot;Georgia&quot;;
	font:bold;
}</string>
      </property>
      <property name="text">
       <string> Word</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_word_add">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border-width: 2px;
	border-style: solid;
	border-color: rgb(69, 70, 70);
	border-radius: 10px;
	height: 17px;

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_desc_add">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border-width: 2px;
	border-style: solid;
	border-color: rgb(69, 70, 70);
	border-radius: 10px;
	height: 17px;

}</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QTextBrowser" name="textBrowser_receivedMessages">
      <property name="styleSheet">
       <string notr="true">QTextBrowser{
	background-color: rgb(255, 255, 255);
	font: bold 13pt  &quot;MS Sans Serif&quot;;
}</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border-width: 2px;
	border-style: solid;
	border-color: rgb(69, 70, 70);
	border-radius: 10px;
	height: 17px;

}</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_word_search">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;background-color: rgb(255, 255, 255);&lt;/p&gt;&lt;p&gt;border-width: 2px;&lt;/p&gt;&lt;p&gt;border-style: solid;&lt;/p&gt;&lt;p&gt;border-color: rgb(69, 70, 70);&lt;/p&gt;&lt;p&gt;border-radius: 10px;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border-width: 2px;
	border-style: solid;
	border-color: rgb(69, 70, 70);
	border-radius: 10px;
	height: 17px;

}</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pushButton_addWord</zorder>
   <zorder>lineEdit_word_add</zorder>
   <zorder>textBrowser_receivedMessages</zorder>
   <zorder>pushButton_searchWord</zorder>
   <zorder>lineEdit_word_search</zorder>
   <zorder>lineEdit_desc_add</zorder>
   <zorder>label_word</zorder>
   <zorder>label_description</zorder>
   <zorder>label_version</zorder>
   <zorder>pushButton_ext_dict</zorder>
   <zorder>lineEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Dictionary</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
